### PR TITLE
fix(wpa): fix & delete pkgs which are removed from winget-pkgs

### PR DESCRIPTION
**Pull request by SpecterShell**
### WinGet Automation Pull Request Checklist

- [x] Have you verified that a previous version of the package is already there in the [Windows Package Manager Community Repository](https://github.com/microsoft/winget-pkgs)?
- [x] Have you tested the package using [`Manage-WpaPackages.ps1 -TestPackage <package-identifier>`](https://github.com/vedantmgoyal2009/vedantmgoyal2009/blob/main/tools/Manage-WpaPackages.ps1) script, provided in the `tools` folder?
- [x] The commit message(s) follows the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines.

### CODE REVIEW

**Comment by vedantmgoyal9** [comment]
Why are you removing redistant packages?

**Comment by SpecterShell** [comment]
These packages have been removed from winget-pkgs because their installers are packed into zip now.

File: src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "PostResponseScript": "$Response = $Response.target",
   "VersionRegex": "[0-9.]+",
   "InstallerRegex": ".(exe|msi|msix|appx)(bundle){0,1}$",
-  "PreviousVersion": "3.9.4.403",
```
Comment (vedantmgoyal9) [inline src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json:15]: The API breaks on setting the version `3.10.10.402`.

File: src/winget-pkgs-automation/packages/t/tencent.start.json [hunk 2]
```diff
@@ -15,8 +15,7 @@
   "PreviousVersion": "0.11.0.9200",
   "ManifestFields": {
     "PackageVersion": "$Response.latestversion",
-    "InstallerUrls": "$Response.downloadurl",
-    "ReleaseDate": "$Response.updatedate.Replace('.', '-')"
```
Comment (vedantmgoyal9) [inline src/winget-pkgs-automation/packages/t/tencent.start.json:19]: Why remove `ReleaseDate`? Does API returns incorrect date?

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

File: src/winget-pkgs-automation/packages/t/tencent.start.json (hunk 2 above)
Comment (SpecterShell) [reply src/winget-pkgs-automation/packages/t/tencent.start.json:19]: Yeah it has been sticked to 2022-05-23 for several versions.

File: src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json (hunk 1 above)
Comment (vedantmgoyal9) [reply src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json:15]: @SpecterShell if this works in your region, I can merge this PR.

**Review by vedantmgoyal9 — APPROVED** [review]

**Comment by vedantmgoyal9** [comment]
@allcontributors please add @SpecterShell for code

File: src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json (hunk 1 above)
Comment (SpecterShell) [reply src/winget-pkgs-automation/packages/t/tencent.tencentmeeting.json:15]: They rolled back to `3.10.10.402` just now, which causes the API returns nothing. At the time I was testing it worked fine.

**Pull request merged**